cfme/tests/infrastructure/test_host.py

```diff
@@ -312,3 +315,30 @@ def _finalize():
             actual_state, power_state
         ),
     )
+
+
+@test_requirements.infra_hosts
+@pytest.mark.meta(coverage=[1738664])
```

Comment:
**Required** since this is a automated test case, you should use `automates` rather than `coverage`

Reply:
Updated as suggested.